libs/sql-schema-describer/tests/introspection_tests.rs

```diff
@@ -47,6 +50,7 @@ fn varchar_full_data_type(api: &TestApi, length: u64) -> String {
         (SqlFamily::Sqlite, _) => format!("VARCHAR({})", length),
         (SqlFamily::Mysql, "mysql8") => format!("varchar({})", length),
         (SqlFamily::Mysql, _) => format!("varchar({})", length),
+        (SqlFamily::Mssql, _) => format!("NVARCHAR({})", length),
```

Comment:
@do4gr you might want to consider this.

`NVARCHAR(n)` contains UCS-2 encoded data, which is predecessor for UTF-16. This means instead of `Vec<u8>`, it has its data in `Vec<u16>`, consuming about double the amount of space compared to UTF-8.

This was the only way of storing non-ASCII characters in SQL Server until SQL Server 2019, which finally allows you to store UTF-8 data to `VARCHAR(n)` columns.

The other thing to understand about varchars is, that the length can be written as a number until `NVARCHAR(4000)` or `VARCHAR(4000)`. After that the column should be a `(N)VARCHAR(max)` that has different storage charasteristics.